qualtran/bloqs/arithmetic_test.py

```diff
@@ -96,5 +105,16 @@ def test_greater_than():
     cbloq = bb.finalize(a=q0, b=q1, result=anc)
 
 
+def test_less_than_equal():
+    lte = LessThanEqual(5, 5)
+    print(lte.decompose_bloq().t_complexity())
```

Comment:
remove print. can this be turned into an assertion?

Reply:
whoops